openedx/frontend-app-authoring
# feat: allow full width content in library authoring [FC-0062]

## Description

This PR allows the library authoring home to go full-width to better fit the content.

### Before:
![image](https://github.com/user-attachments/assets/32ef31d2-17bf-408e-b8f1-19b99a719a28)

### After:
![image](https://github.com/user-attachments/assets/f2f374ca-9c96-4373-8d57-93f4bf4bf347)

### Responsiveness:
![library-authoring-responsiveness](https://github.com/user-attachments/assets/14717d08-3675-4b09-9edb-bc79e5746f52)

## More information
Part of:
- https://github.com/openedx/frontend-app-course-authoring/issues/1221

Depends on:
- https://github.com/openedx/paragon/pull/3216
- https://github.com/openedx/frontend-component-footer/pull/463
- https://github.com/openedx/frontend-component-header/pull/529

## Testing instructions
- Checkout this branch and run `npm install --legacy-peer-deps` to ignore the peer-deps conflicts and make sure you have the correct dependencies
- If you experience `Module not found` webpack errors, it might be due to dependency cache issues. Run `npm run build`, and restart tutor.
- Open the library authoring page for a library with some components
- Resize the screen and check the responsiveness
___
Private ref: [FAL-3820](https://tasks.opencraft.com/browse/FAL-3820)
